sdk/rust/oak_utils/src/tests.rs

```diff
@@ -0,0 +1,70 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+use crate::*;
+
+use std::fs::File;
+use std::io::Write;
+use std::path::Path;
+
+fn write_files(dir: &Path, filenames: &[&str], data: &[&str]) {
+    for (filename, data) in filenames.iter().zip(data.iter()) {
+        let path = dir.join(filename);
+        let mut file = File::create(path).unwrap();
+        file.write_all(data.as_bytes()).expect("Write error");
+    }
+}
+
+const TEMP_FILES: [&str; 3] = ["1", "2", "3"];
```

Comment:
Could these be defined as slices rather than arrays? Something like

`const TEMP_FILES: &[&str] = &["1", "2", "3"];`

So we don't need to specify the size every time.